### DISCUSSION TITLE
How to use fortls python api

### DISCUSSION THREAD

**Discussion by vickysharma0812** [fortran-lang/fortls#259] [Q&A]
Hello everyone,

For documentation of my source code, I want to extract the interfaces of routines  (symbols) included inside a module source file. Is it possible to perform this task by using fortls python API?  

Regards
Vikas

**Comment by gnikit** [comment]
I don't think fortls is the best tool for this job. I think ford is much better suited for this https://github.com/Fortran-FOSS-Programmers/ford

**Reply by vickysharma0812** [reply]
Thanks, I agree with you!
